OpenRA.Mods.Common/Traits/World/PathSearch.cs

```diff
@@ -331,10 +348,10 @@ CellLayer<CellInfo> InitCellInfo()
 					defaultCellInfoLayer.Size != mapSize ||
 					defaultCellInfoLayer.Shape != map.TileShape)
 				{
-					defaultCellInfoLayer = new CellLayer<CellInfo>(map);
+					defaultCellInfoLayer = new CellLayer<CellInfo>(map.TileShape, mapSize);
 					for (var v = 0; v < mapSize.Height; v++)
 						for (var u = 0; u < mapSize.Width; u++)
-							defaultCellInfoLayer[new MPos(u, v)] = new CellInfo(int.MaxValue, new MPos(u, v).ToCPos(map), false);
+							defaultCellInfoLayer[new MPos(u, v)] = new CellInfo(int.MaxValue, new MPos(u, v).ToCPos(map.TileShape), false);
```

Comment:
Just avoiding an unnecessary inter-call. I don't think the intercall is justified.